src/components/messenger/list/group-details-panel/index.tsx

```diff
@@ -58,7 +65,12 @@ export class GroupDetailsPanel extends React.Component<Properties, State> {
             onChange={this.nameChanged}
             {...cn('name-input')}
             placeholder='Group name...'
+            isRequired={true}
+            aria-invalid={!this.state.isNameValid}
           />
+          {!this.state.isNameValid && (
+            <div style={{ color: 'rgba(221, 50, 50, 0.75)' }}>Please enter a group name.</div>
+          )}
```

Comment:
I think you should just be able to use the `error` & `errorMessage` field in the `<Input>` component itself. We don't need `area-invalid` & an extra state.

Reply:
Thanks, I’m going to change it to this component!

Reply:
Yep as Ratik suggested. Also, I'm seeing a number of inline styles being added on tasks. We should be using the scss stylesheets to set css, not the `style` attribute on elements. Thanks!

Reply:
good tip

Reply:
@ratik21 @domw30 ref here: 
#9547793